.github/workflows/deploy-pypi.yml

```diff
@@ -56,3 +56,30 @@ jobs:
         with:
           name: ${{ env.INPUT_NAME }}
           tag_name: ${{ env.INPUT_NAME }}
+  spaces-test-release:
```

Comment:
Ran this on my fork here: https://github.com/freddyaboulton/gradio/pull/7

You can see the demo here: https://huggingface.co/spaces/gradio-test-deploys/3.1.4_kitchen_sink

Reply:
The Space is broken right now. Is that because of the Dataframe example caching?